src/common/options.cc

```diff
@@ -8954,7 +8954,7 @@ std::vector<Option> get_mds_client_options() {
     .set_default(false)
     .set_description("issue new requests to a random active MDS"),
 
-    Option("client_mount_timeout", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
+    Option("client_mount_timeout", Option::TYPE_SECS, Option::LEVEL_ADVANCED)
     .set_default(300.0)
```

Comment:
Here maybe you need to fix it.

`s/300.0/300/`

Reply:
It seems this will cause the `ceph API tests` failure.

Reply:
Done!!